### ISSUE TITLE
Inconsistency between scans of the same single endpoint

### ISSUE THREAD

**Issue by Autodater** [zaproxy/zaproxy#6110]
I have executed three identical scans targeting the same single endpoint and in all three cases the outcome was different

1. Crawl the site using a proxy to create a site map in Zap 
2. Once done, select a request in the History tab: POST */vinLookup.ajax? that has 7 parameters in its body (see the body in **Screenshot 1**)
3. Right-click and select Attack>Active scan
4. Navigate to the Policy tab of the Active Scan window and make the desired changes (**Screenshot 2**)
5. Leave everything else to default
6. Start the scan of this endpoint
7. See the results of the finished scan in **Screenshot 3**
8. Delete these two logged items in the SQL Injection sub folder in the Alerts tab
9. Go about executing the same scan and once it is finished see its outcome in **Screenshot 4**
10. Delete these three logged items in the SQL Injection sub folder in the Alerts tab
11. Go about executing the same scan and once it is finished see its outcome in **Screenshot 5**

Actual scan results:  the outcome of all scans is different, with different parameters logged by each scan

Expected result: the outcome of all scans should be the same

P.S. I was asked to re-do the scan having set the thread concurrency to 1, which I did in the Options>Active Scan window.
The outcome of this scan was different altogether, meaning none of SQL injection threats were logged.

**Screenshot 1**
![screenshot-1](https://user-images.githubusercontent.com/69215155/89335919-aa46db00-d666-11ea-92b8-a10b2f5f3dd0.PNG)

**Screenshot 2**
![screenshot-2](https://user-images.githubusercontent.com/69215155/89333017-5cc86f00-d662-11ea-95e9-e7803d16c6de.PNG)

**Screenshot 3**
![screenshot-3](https://user-images.githubusercontent.com/69215155/89335488-183ed280-d666-11ea-81b5-0107f2ffabc7.PNG)

**Screenshot 4**
![screenshot-4](https://user-images.githubusercontent.com/69215155/89333729-669ea200-d663-11ea-9a69-c8d95b3ef659.PNG)

**Screenshot 5**
![screenshot-5](https://user-images.githubusercontent.com/69215155/89333998-d876eb80-d663-11ea-994c-1933ca7e7a54.PNG)

**Software versions**
 - ZAP: 2.9.0
 - Add-on: default plugins
 - OS: Windows 10
 - Java: JDK 9.0.4
 - Browser: n/a

I will provide the logs if they are required

**Comment by kingthorin** [comment]
Could you set scan threads to 1 and see what happens?
https://www.zaproxy.org/docs/desktop/ui/dialogs/options/ascan/#concurrent-scanning-threads-per-host

**Comment by Autodater** [comment]
Done. This time around none was logged whatsoever.

**Comment by kingthorin** [comment]
Thanks for confirming, this seems to be a duplicate of: https://github.com/zaproxy/zaproxy/issues/3662
